src/utils/image.js

```diff
@@ -251,3 +136,118 @@ export async function getImages() {
 	images.sort((a, b) => b.date - a.date);
 	return images;
 }
+
+export async function uploadImages(
+	files,
+	Toast,
+	setNoPick,
+	setProgress,
+	setUploading,
+	setImages,
+	next,
+	resolve
+) {
+	const host = await getHostSettings();
+	const settings = await getSettings();
+	/* eslint-disable no-nested-ternary */
+	const url =
+		host === aHosts[1]
+			? settings.apiUrl
+			: host === aHosts[0]
+			? 'https://api.imgbb.com/1/upload'
+			: 'https://api.imgur.com/3/image';
```

Comment:
for the love of mother nature - and my sanity, (reminder hosts should be obj with named keys)
```suggestion
let url;
switch (host) {
	case aHosts[1]:
		url = settings.apiUrl;
		break;
	case aHosts[0]:
		url = 'https://api.imgbb.com/1/upload';
		break;
	default:
		url = 'https://api.imgur.com/3/image';
```